docs/docs/recipes/manage-users/on-admin-console-page.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: Manage on Admin Console page
+sidebar_position: 1
+---
+
+# Manage users on the Admin Console page
+
+import GotoUserManagement from './fragments/_goto-user-management.md';
+
+## List users
+
+<GotoUserManagement />
+
+There are three columns in the table:
+
+- **User**: The user info includes avatar, name (for display), and username (for sign-in).
+- **From application**: The name of the application from which the user is.
+- **Latest sign in**: The timestamp when the user signed in last time.
+
+To navigate users, click the "<" previous page button, the ">" next page button, or the specific page number at the bottom right corner.
+
+## Search users
+
+<GotoUserManagement />
+
+Input a keyword in the search box above the user table, click the "Search" button, and you'll see the matching users in the table.
+
+The keyword can be:
+
+- name,
+- username,
+- email,
+- or phone number.
```

Comment:
Fuzzy search is supported on the following user properties:

- name
- username
- email
- phone number

Reply:
exact and partial match for now.

Reply:
Updated. PTAL

```md
Support the exact partial search on the following user properties:
```